docs/maps/DRO.json

```diff
@@ -100,6 +100,10 @@
               "description": "Access level for the DRO when released from embargo.",
               "type": "string",
               "enum": ["world", "stanford", "location-based", "citation-only", "dark"]
+            },
+            "useAndReproductionStatement": {
+              "description": "The human readable reuse and reproduction statement that applies to when released from embargo.",
```

Comment:
`that applies to when released from embargo` -> `that applies to a DRO when released from embargo`?

Reply:
or maybe: ... "that applies when released from embargo"